quickwit/quickwit-metastore/src/metastore/file_backed/lazy_file_backed_index.rs

```diff
@@ -86,19 +86,30 @@ impl LazyFileBackedIndex {
 async fn poll_index_metadata_once(
     storage: &dyn Storage,
     index_id: &str,
-    metadata_mutex: &Mutex<FileBackedIndex>,
+    unlocked_index: &Mutex<FileBackedIndex>,
```

Comment:
it is not unlocked? I don't understand the name of that variable.

Reply:
Will change to something I'm not the only one to understand :)